components/monitoring/memory/memory.html

```diff
@@ -28,6 +28,9 @@
         <canvas id="canvasMemUsage"></canvas>
     </div>
 </div>
+
+<h3>Banks</h3>
```

Comment:
The bank list is empty when the SI API is accessed without root on linux.
Please fix this header to only appear when needed.
![image](https://user-images.githubusercontent.com/1696674/51442156-4ce47680-1cda-11e9-89dc-0cb6af95ee59.png)
Example included ^